parlai/crowdsourcing/tasks/dialcrowd/run.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from dataclasses import dataclass, field
+from typing import Any, List
+
+import hydra
+from mephisto.operations.hydra_config import register_script_config
+from omegaconf import DictConfig
+
+from parlai.crowdsourcing.tasks.dialcrowd.dialcrowd_blueprint import (
+    STATIC_BLUEPRINT_TYPE,
+)
+from parlai.crowdsourcing.tasks.dialcrowd.util import run_static_task
+from parlai.crowdsourcing.utils.mturk import MTurkRunScriptConfig
+
+
+TASK_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+
+
+defaults = [
```

Comment:
nit: we've been centralizing Hydra defaults in the yaml files as opposed to the run.py files for consistency. See https://github.com/facebookresearch/ParlAI/blob/main/parlai/crowdsourcing/tasks/acute_eval/hydra_configs/conf/base_fast_acute.yaml for instance